content/docs/04-clusters/06-cluster-management/11.5-kubeconfig.md

```diff
@@ -0,0 +1,99 @@
+
+# Kubeconfig Access Permissions
+
+Palette exposes the kubeconfig file for each cluster deployed through Palette. Depending on the cluster's configuration, the kubeconfig file may contain different configurations, such as the cluster's API server address and authentication credentials. 
+
+Your assigned [Palette permissions](/user-management/palette-rbac/project-scope-roles-permissions) determine which clusters you can access and what operations you can perform on the cluster. The permissions assigned to you in Palette determine if you can download and access the kubeconfig file for a cluster. 
+
+As a rule of thumb, users with the Palette project role [*Cluster Admin*](/user-management/palette-rbac/project-scope-roles-permissions#cluster) can access the kubeconfig file for all clusters in the project. Users with lower-level project roles such as the  *Cluster Editor* or the *Cluster Viewer* may not be able to access the kubeconfig file of the cluster. 
+
+<br />
+
+<InfoBox>
+
+Palette has its own RBAC system that is separate from the [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) system in the host cluster. The permissions assigned to you in Palette determine what operations you can perform on the cluster. The permissions assigned to you in the host cluster through the Kubernetes RBAC system determine what operations you can perform inside the cluster and on its resources. Refer to the [Palette Roles and Kubernetes Roles](/clusters/cluster-management/cluster-rbac#paletterolesandkubernetesroles) for additional information.
+
+</InfoBox>
+
+
+The access you have as a user to the kubeconfig file for a cluster depends on the following factors:
+
+<br />
+
+* Is OIDC configured for the cluster? OIDC is configured in the Kubernetes pack YAML file of the cluster profile. Refer to the respective Kubernetes distribution [Pack reference page](/integrations) to learn more about OIDC configuration.
+
+  <br />
+
+  <WarningBox>
+
+    When enabling OIDC, ensure the parameter `oidc-issuer-url` and the `clientConfig` block are properly configured in the Kubernetes Pack YAML. Properly configuring both parameters, you ensure the kubeconfig file is available for all project users. Otherwise, the kubeconfig file will only be available for Cluster Admins or custom roles that have the *delete* permission for the resource key *cluster*.
+
+  </WarningBox>
+
+
+* Is the [Spectro Proxy](/integrations/frp) enabled for the cluster?
+
+
+Use the following tables to help you identify what project role you need to access the kubeconfig file for a cluster. 
+
+
+### Cluster Admin
+
+If a custom Palette resource role is created and used instead of the Palette role *Cluster Admin*. The custom Palette resource role must have the *delete* permissions for the resource key *cluster* to access the kubeconfig file for a cluster.
```

Comment:
The first sentence is incomplete. Or is it intended to be joined with the second?

Also, do you want to use 'you'? "If you use a custom Palette resource role..."